.coveragerc

```diff
@@ -1,5 +1,5 @@
 [run]
-include = sprint_management_application/*
```

Comment:
@Agrendalath, sorry about requesting a name change. It is a huge change and a lot of work to test and verify that everything works okay.

Reply:
No worries, `find -type f -print0 |xargs -0 sed -i 's/sprint_management_application/sprints/g'` + a little manual work around IDE configs didn't take much time.